src/OI.cpp

```diff
@@ -22,59 +22,62 @@
 OI::OI() {
     // Process operator interface input here.
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
-    coDriver.reset(new Joystick(1));
     
-    coLGreen.reset(new JoystickButton(coDriver.get(), 15));
+	coDriver.reset(new Joystick(1));
+
+    coLGreen.reset(new JoystickButton(coDriver.get(), kLGreen_ID));
     coLGreen->WhileHeld(new AutonomousCommand());
-    coLRed.reset(new JoystickButton(coDriver.get(), 12));
+    coLRed.reset(new JoystickButton(coDriver.get(), kLRed_ID));
```

Comment:
`kLRed`? Does that stand for `kLeftRed`? 

Avoid abbreviations (that aren't totally obvious).

No change required.